src/table/paginator.scss

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import '../button/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 100%;
+    flex: 1;
+  }
+
+  > .iui-center {
+    justify-content: center;
+    @include themed {
+      color: t(iui-text-color);
+    }
+
+    .iui-button-group {
+      margin: 0 $iui-s;
+
+      > .iui-button,
+      > .iui-ellipsis {
+        width: $iui-button-height;
+
+        &.iui-small {
+          width: $iui-button-height-small;
+        }
+      }
+
+      > .iui-ellipsis {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: default;
+        @include themed {
+          color: t(iui-text-color-muted);
+        }
+      }
+
```

Comment:
Maybe our active invisible button should have had it always at the bottom then.. :(

Reply:
I think in most cases the buttons are placed above the content.  It's only because the paginator buttons are underneath the content that I think placing the lines on the bottom looks better.